testing/vcs/test_script_for_py.py

```diff
@@ -0,0 +1,15 @@
+import vcs
+
+list=["boxfill","meshfill","isofill","textcombined","texttable","vector"]
+for obj in list:
+    s=''
+    tc=''
+    if obj == 'textcombined':
+        vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left')
+        tc="'EXAMPLE_tt','EXAMPLE_tto'"
+    code="s=vcs.get%s(%s)" % (obj, tc)
+    exec(code)
+    try:
+        s.script(obj+"_script.py")
```

Comment:
You need to delete the files after you generate them. Also, this only builds the script; for this to be a real test, it needs to validate correctness. That means running the scripts and seeing if the generated object matches the original.